dask_geopandas/core.py

```diff
@@ -163,7 +162,7 @@ def set_crs(self, value, allow_override=False):
         return new
 
     def to_crs(self, crs=None, epsg=None):
-        token = f"{self._name}-to_crs"
+        token = "to_crs"
         return self.map_partitions(M.to_crs, crs=crs, epsg=epsg, token=token)
```

Comment:
I'm not sure this is entirely necessary. I would expect that using the name of the method is the default dask behavior, however if we want to be explicit that is fine with me.

Reply:
Indeed not needed here!